frontend/app/src/locales/en.json

```diff
@@ -89,6 +108,29 @@
       "password_missmatch": "The password confirmation does not match the provided password"
     }
   },
+  "conflict_dialog": {
+    "action": {
+      "local": "Local",
+      "remote": "Remote"
+    },
+    "all_buttons_description": "For All conflicts",
+    "buttons": {
+      "cancel": "Cancel",
+      "confirm": "Confirm"
+    },
+    "subtitle": "Local changes have been detected to the following assets. Select if you would like to keep the local version or overwrite with the remote.",
+    "table": {
+      "headers": {
+        "local": "Local",
+        "remote": "Remote",
+        "keep": "Keep"
+      }
+    },
+    "title": "Conflicts detected",
+    "keep_local": "Keep Local",
+    "keep_remote": "Keep Remote",
+    "hint": "{conflicts} asset entries cannot be automatically updated. {remaining} remaining entry/ies require user input."
```

Comment:
Is it possible to know the number and change this to entry or entries respectively? If not I would just stick with entries.

Reply:
I am not sure if I can use plurals with the i18n component I have to check. if it was a normal thing this would be possible

Reply:
Yeah I guessed it may be tricky. using plural for `1` is also totally fine

Reply:
I did check when using this i18n component for interpolation unfortunately it is not possible to use plurals in this case.